policyengine_us/variables/gov/states/co/hcpf/chp/co_chp_out_of_pocket_maximum.py

```diff
@@ -0,0 +1,13 @@
+from policyengine_us.model_api import *
+
+
+class co_chp_out_of_pocket_maximum(Variable):
+    value_type = float
+    entity = Person
+    label = "Colorado Child Health Plan Plus out of pocket maximum"
+    definition_period = YEAR
+
+    def formula(person, period, parameters):
+        income_level = person.tax_unit("medicaid_income", period)
```

Comment:
income_level refers to % of poverty right?
```suggestion
        income = person.tax_unit("medicaid_income", period)
```